docs/index.html

```diff
@@ -948,7 +922,7 @@ <h4 class="headline-3">Module: 1:1:1</h4>
 				<h4 class="headline-3">Module: Grid - [2 : [1,1]] / [[1,1] : 2]</h4>
 				<p>
 					We can also wrap grid-cells together, so they will flow with the col, but not with the row what their usual behaviour is. <br />
-					The <code>element-wrapper</code> class can help us here. 
+					The <code>grid-nested</code> class can help us here. BUT - the nested elements only work if the Screen has a min-width of 665px. 
```

Comment:
Here I'm not sure if having a fixed value makes sense as it could change. Maybe refer to the CSS var name and add a reference like https://github.com/webcompat/design/blob/master/docs/css/variables.css#L17?

Reply:
no fix value and it's not like it's not working, it's only defined at the moment from 665 on